File: src/main/java/gg/mineral/practice/util/messages/UsageMessage.java
package gg.mineral.practice.util.messages;

import org.bukkit.entity.Player;

import lombok.val;

public class UsageMessage extends Message {

	public UsageMessage(String s) {
		message = s;
		formatMessage();
	}

	private void formatMessage() {
		val addition = CC.D_RED + "Usage: " + CC.RED;
		message = addition + message + ".";
	}

	@Override
	public void send(Player p) {
		p.sendMessage(message);
	}
}

File: src/main/kotlin/gg/mineral/practice/util/messages/UsageMessage.kt
package gg.mineral.practice.util.messages

class UsageMessage(s: String) : Message(s) {
    override fun clone() = UsageMessage(messageBuilder.toString())

    override fun prepend(builder: StringBuilder): StringBuilder =
        messageBuilder.insert(0, CC.D_RED + "Usage: " + CC.RED).append(".")
}
